bin/varnishncsa/varnishncsa.c

```diff
@@ -786,15 +794,24 @@ dispatch_f(struct VSL_data *vsl, struct VSL_transaction * const pt[],
 	unsigned tag;
 	const char *b, *e, *p;
 	struct watch *w;
-	int i, skip;
+	int i, skip, be_mark;
 	(void)vsl;
 	(void)priv;
 
+#define BACKEND_MARKER VSL_t__MAX
```

Comment:
I liked this better when BACKEND_MARKER was waaay higher than VSL_t__MAX. If the varnishlog and varnishd binaries are out of sync (not completely unlikely if e.g. one considers a log sent to support), then there can be strangeness observed.